src/rest-server/src/config/paiConfig.js

```diff
@@ -26,18 +26,19 @@ try {
     paiMachineList = yaml.safeLoad(fs.readFileSync('/pai-cluster-config/layout.yaml', 'utf8'))['machine-list'];
 } catch (err) {
     paiMachineList = [];
-    logger.info('Unable to load machine list from cluster-configuration.');
-    logger.info(err.stack);
+    logger.error('Unable to load machine list from cluster-configuration.');
```

Comment:
Since it's not a fatal error in this process, I think `warn` is better, also add details of what it'll cause.

Reply:
Done